src/shared/components/instances-definitions.ts

```diff
@@ -114,6 +114,11 @@ export const RECOMMENDED_INSTANCES: RecommendedInstance[] = [
     languages: ["en"],
     topics: [GENERAL, TECHNOLOGY, POLITICS],
   },
+  {
+    domain: "toast.ooo",
+    languages: ["en"],
+    topics: [GENERAL, TECHNOLOGY, GAMING]
```

Comment:
You missed a comma here, so its failing lint.

Reply:
Whoops, I’ll go ahead an fix that